src/components/Category.js

```diff
@@ -1,7 +1,14 @@
-const Category = () => (
-  <div>
-    <button type="button">Check status</button>
-  </div>
-);
+import { useSelector, useDispatch } from 'react-redux/es/exports';
+import { CheckStatus } from '../redux/categories/categories';
 
+const Category = () => {
+  const response = useSelector((state) => state.categoriesReducer);
```

Comment:
- [ ] When I click the check status button "Under construction" is not displayed on the page. This is because you are trying to access the wrong reducer. If you check your store configuration, you'll see that you reassigned the `categoriesReducer` to `cateogories`, so this should be something like so:

  ```
  const response = useSelector((state) => state.categories);
  ```